lab/src/Microsoft.ServiceModel.Syndication/src/Resources/Resources.txt

```diff
@@ -395,7 +395,7 @@ OperationPropertyIsRequiredForAttributeGeneration=Operation property of Operatio
 InvalidMembershipProviderSpecifiedInConfig=The username/password Membership provider {0} specified in the configuration is invalid. No such provider was found registered under system.web/membership/providers.
 InvalidRoleProviderSpecifiedInConfig=The RoleProvider {0} specified in the configuration is invalid. No such provider was found registered under system.web/roleManager/providers.
 ObjectDisposed=The {0} object has been disposed.
-InvalidReaderPositionOnCreateMessage=The XmlReader used for the body of the message must be positioned on an element.
+InvalidReaderPositionOnCreateMessage=The XmlReaderWrapper used for the body of the message must be positioned on an element.
```

Comment:
>XmlReaderWrapper [](start = 41, length = 16)

Probably want to keep this as XmlReader as we are making calls on the same XmlReader